src/components/Mechanism/WordsOrImage.jsx

```
@@ -1069,64 +1081,80 @@ const WordsOrImage = ({
+                    <Box
                       sx={{
-                        fontSize:
-                          language === "te"
+                        // On mobile: constrain height and enable scroll so long
+                        // paragraphs can never overlap the diamonds header zone
+                        maxHeight: { xs: "calc(100dvh - 260px)", md: "none" },
+                        overflowY: { xs: "auto", md: "visible" },
+                        width: "100%",
+                        WebkitOverflowScrolling: "touch",
+                      }}
+                    >
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)